src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="header__logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
```suggestion
        <ul class="nav__list">
```
follow bem naming convention
https://en.bem.info/methodology/naming-convention/#naming-rules